connection.js

```diff
@@ -713,20 +712,16 @@ function get_r_u_arr_booleans(familyNameString, notificationRanked) {
 
 function getBooleans_r_u_Reject(argNameString, nameF, ending) {
     const rankedUnranked = beforeRankedUnrankedGamesSpecial("for ", "", argNameString, "");
-    const msg = `${nameF} not allowed on this bot ${rankedUnranked}${ending}.`;
-    conn_log(msg);
-    return { reject: true, msg };
+    return getReject(`${nameF} not allowed on this bot ${rankedUnranked}${ending}.`);
```

Comment:
Do not forget to add unit testing on these changes. I dont think they are covered.

Reply:
i'd like to do that, but would need time to get used to it, i definitely will try to do it sometime :100: